equinix/resource_fabric_service_profile_acc_test.go

```diff
@@ -17,139 +17,134 @@ import (
 	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
 )
 
-func testAccFabricReadServiceProfileConfig(uuid string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profile" "test" {
-	uuid = "%s"
-	}
-`, uuid)
-}
-
-func testAccFabricReadServiceProfilesListConfig(name string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profiles" "test" {
-		filter {
-			property = "/name"
-			operator = "="
-			values = ["%s"]
-		}
-	}
-`, name)
-}
-
-func TestAccFabricReadServiceProfileByUuid(t *testing.T) {
+func TestAccFabricCreateServiceProfile_SP_FCR_ba(t *testing.T) {
```

Comment:
All test names need to end in `_PFCR` or `_PNFV` depending on which user they have so that our test running will only run fabric tests with the correct suffix.

Reply:
done